sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/GatewayConnectionConfig.java

```diff
@@ -40,23 +40,23 @@ public static GatewayConnectionConfig getDefaultConfig() {
     }
 
     /**
-     * Gets the request timeout (time to wait for response from network peer).
+     * Gets the network request timeout (time to wait for response from network peer).
      *
-     * @return the request timeout duration.
+     * @return the network request timeout duration.
      */
-    Duration getRequestTimeout() {
-        return this.requestTimeout;
+    Duration getNetworkRequestTimeout() {
+        return this.networkRequestTimeout;
     }
 
     /**
-     * Sets the request timeout (time to wait for response from network peer).
-     * The default is 5 seconds.
+     * Sets the network request timeout (time to wait for response from network peer).
+     * The default is 60 seconds.
      *
-     * @param requestTimeout the request timeout duration.
+     * @param networkRequestTimeout the network request timeout duration.
      * @return the {@link GatewayConnectionConfig}.
      */
-    GatewayConnectionConfig setRequestTimeout(Duration requestTimeout) {
-        this.requestTimeout = requestTimeout;
+    GatewayConnectionConfig setNetworkRequestTimeout(Duration networkRequestTimeout) {
```

Comment:
We shouldn't allow customer to set gateway timeout to less than 60 seconds. This is the required because if a split happens the gateway needs time to retry to handle the split. Allowing less than 60 seconds may cause issues for the gateway because it will keep retrying request that the gateway is already trying to complete. This limitation is in the .NET SDK.

Reply:
Currently, this value is not used by java sdk. 
Internally the http requestTimeout is by default 60s or can be override by system property. 

Will add a checking here in case this will be used in the future.

Reply:
added the config validation.